src/coreComponents/common/format/StringUtilities.cpp

```diff
@@ -93,7 +93,9 @@ string addCommaSeparators( T const & num )
   return result;
 }
 
+#if GEOS_LOCALINDEX_TYPE_FLAG != GEOS_GLOBALINDEX_TYPE_FLAG
 template string addCommaSeparators( localIndex const & num );
+#endif
 template string addCommaSeparators( globalIndex const & num );
```

Comment:
I think this should be better

```suggestion
template string addCommaSeparators( int const & num );
template string addCommaSeparators( long int const & num );
template string addCommaSeparators( long long int const & num );
```